src/Microsoft.Azure.SignalR.Common/ServiceConnections/ServiceConnectionBase.cs

```diff
@@ -637,16 +631,16 @@ private static class Log
         {
             // Category: ServiceConnection
             private static readonly Action<ILogger, ulong?, string, string, Exception> _failedToWrite =
-                LoggerMessage.Define<ulong?, string, string>(LogLevel.Error, new EventId(1, "FailedToWrite"), "Failed to send message {tracingId} to the service: {message}. Id: {ServiceConnectionId}");
+                LoggerMessage.Define<ulong?, string, string>(LogLevel.Error, new EventId(1, "FailedToWrite"), "Failed to send message {TracingId} to the service. Error detail: {Error}. Id: {ServiceConnectionId}");
 
             private static readonly Action<ILogger, string, string, string, Exception> _failedToConnect =
-                LoggerMessage.Define<string, string, string>(LogLevel.Error, new EventId(2, "FailedToConnect"), "Failed to connect to '{endpoint}', will retry after the back off period. Error detail: {message}. Id: {ServiceConnectionId}");
+                LoggerMessage.Define<string, string, string>(LogLevel.Information, new EventId(2, "FailedToConnect"), "Failed to connect to '{Endpoint}', will retry after the back off period. Error detail: {Error}. Id: {ServiceConnectionId}");
```

Comment:
we define "Warning" (maybe caused by service reload, timeout, ..., so we use "Warning") in service, should we keep the same?

Reply:
I feel there is nothing to warn the customer since it does not have real impact